src/v2/Components/Onboarding/ItemLink.tsx

```diff
@@ -48,6 +48,10 @@ interface Props extends React.HTMLProps<HTMLAnchorElement> {
   id: string
   name: string
   image_url: string
+  onFollow: (
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
```

Comment:
Question: does this type need to be tied to the mouse? I wonder if there's a more general type that would also work? 🤔

Reply:
I just left the initial type for click event as the 'onClick' method passes its event to the 'onFollow' method. But I've just realized that 'onFollow' method doesn't use the event object at all and I should've removed it from parameters.